.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Do you want to remove the vscode settings if you're using IntelliJ now (as is everyone else on the team I think)?